en/sensor/rangefinders.md

```diff
@@ -42,6 +42,8 @@ The rangefinders are enabled using the parameter [SENS_EN_MB12XX](../advanced_co
 PX4 supports: SF11/c and SF/LW20.
 PX4 can also be used with the following discontinued models: SF02, SF10/a, SF10/b, SF10/c.
 
+PX4 also supports the [LightWare LiDAR SF45 Rotating Lidar](https://www.lightwarelidar.com/shop/sf45-b-50-m/) for [collision prevention](../computer_vision/collision_prevention.md#lightware-lidar-sf45-rotating-lidar) applications.
```

Comment:
@dirksavage88 I also added a link to the lidar and our docs here. Would this lidar ever be used for "general use" or is it really just for the collision prevention/obstacle avoidance use case?

What I'm thinking is whether I need to have a peer of https://docs.px4.io/main/en/sensor/rangefinders.html for scanning rangefinders or similar and move some of the setup into that. Just a thought - won't block merging this.

Reply:
It’s only for collision prevention. There is a ROS driver out there but for the px4 avoidance package it probably won’t work out of the box